Example/Array+MiniApp.swift

```diff
@@ -0,0 +1,14 @@
+extension Array {
+    public func dictionnaryFilteredBy<Key: Hashable> (index selectKey: (Element) -> Key) -> [Key: [Element]] {
```

Comment:
thanks for the typo, French go me again. Yeah, it could be a good idea to keep this kind of helpers, but not until we have the need. If we implement a common utils library it would go inside it.